k8s/configMap-sjc.yaml

```diff
@@ -78,3 +78,6 @@ data:
       - Id: services-all
         FrontendRegexp: ^services\.wikia\.com
         Sampling: 0.5
+      - Id: mobile-wiki:
+        FrontendRegexp: ^mobile-wiki-[^\.]+\.prod\.sjc\.k8s\.wikia\.net
```

Comment:
I think you can just use the actuall ingress name here, `[^\.]+` is not necessary